### PR TITLE
governance page updates

### PR DESCRIPTION
**Pull request by JosephMcArthur**
See https://github.com/oaworks/internal-planning/issues/375 for discussion

### CODE REVIEW

**Comment by sssoz** [comment]
We should also set up a redirect for the old transparency page

**Comment by JosephMcArthur** [comment]
Ah, yes! Can you set that up please.

Joe McArthur
Director / Co-Founder: OA.Works <https://oa.works>
The Open Access Button is now OA.Works! Find out more about our rebrand
<https://blog.oa.works/open-access-button-is-now-oa-works/>.

**Comment by JosephMcArthur** [comment]
Not yet –– it's not done copy wise

Joe McArthur
Director / Co-Founder: OA.Works <https://oa.works>
The Open Access Button is now OA.Works! Find out more about our rebrand
<https://blog.oa.works/open-access-button-is-now-oa-works/>.

File: src/about/governance/index.md [hunk 1]
```diff
@@ -5,35 +5,35 @@ title: Governance
...
+None of our Advisory Committee members are paid. However, we are able to provide compensation of up to 49% of the Advisory Committee costs should it enable people from underrepresented groups to participate.
 
 ## Articles of Governance
 
 Articles of Governance set the rules for how organizations like ours are run. They’re similar to “bylaws”, but we don’t use that term to avoid confusion with Code for Science & Society’s bylaws. “Articles” are just sections that detail some aspect of our governance.
 
-Our [Articles of Governance](/about/governance/articles-of-governance) define the overall structure of OA.Works, its leadership, and critical processes. Ours focuses on defining the Advisory Committee’s powers, voting as its mechanism for using those powers, and creating inclusive rules for how the committee itself is composed and run.
+Our [Articles of Governance](/about/governance/articles-of-governance) define the overall structure of OA.Works, its leadership, and critical processes. Ours focus on defining the Advisory Committee’s powers, voting as its mechanism for using those powers, and creating inclusive rules for how the committee itself is composed and run.
```
Comment (sssoz) [inline src/about/governance/index.md:30]: It’s "Ours focuses on..."

Comment (sssoz) [reply src/about/governance/index.md:30]: DISREGARD!

[the remark this answers is not in the document]
Comment (JosephMcArthur) [reply src/about/governance/index.md:30]: The Articles of Governance. I think I can actually remove a bit of redundancy here left from an earlier draft!

**Pull request merged**